Specs/Number/Turkish/OrdinalModel.json

```diff
@@ -303,5 +303,124 @@
     "Input": "10.000tl",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": []
+  },
+  {
+    "Input": "1. sırada",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "1.",
+        "Start": 0,
+        "End": 1,
+        "TypeName": "ordinal",
+        "Resolution": {
+          "offset": "1",
+          "relativeTo": "start",
+          "value": "1"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "önceki",
```

Comment:
Adding some short entity focus spec cases is fine, but in future changes let's also try to add some examples of full sentences, and sentences that mention multiple entities to help guarantee there are no unexpected side-effects for changes, ok?

Reply:
ok, we will do it